src/test/java/org/folio/rest/api/TestBase.java

```diff
@@ -106,6 +107,7 @@ public static void testBaseBeforeClass() throws Exception {
     instanceRelationshipsClient = ResourceClient.forInstanceRelationships(client);
     instanceRelationshipTypesClient = ResourceClient.forInstanceRelationshipTypes(client);
     precedingSucceedingTitleClient = ResourceClient.forPrecedingSucceedingTitles(client);
+    boundWithPartsClient = ResourceClient.forBoundWithParts(client);
```

Comment:
Is this used in any tests other than src/test/java/org/folio/rest/api/BoundWithStorageTest.java?

If not, I suggest adding it that class instead of the base class.

Reply:
Was moved away from base class after your review.

Reply:
Thank you